### PR TITLE
Added stubs for Arbitrary File & Content View on API/CLI

### LINKED ISSUE #5052: Create Stubs for Arbitrary File Repository & Content View on API/CLI

### PR DESCRIPTION
**Pull request by renzon**
close #5052

### CODE REVIEW

File: tests/foreman/api/test_contentview.py [hunk 1]
```diff
@@ -1807,3 +1807,102 @@ def test_positive_publish_promote_with_RH_ostree_and_other(self):
...
+        Content View
+
+        :id: 133c4a68-ed5a-477a-ae62-8b9a3703ae6b
+
+        :Setup:
+            1. Create a File Repository (FR)
+            2. Upload an arbitrary file to it
+            3. Create a Content View (CV)
+
+        :Steps:K
```
Comment (rochacbruno) [inline tests/foreman/api/test_contentview.py]: an extra `K` ?

File: tests/foreman/api/test_repository.py [hunk 2]
```diff
@@ -59,7 +59,7 @@
     skip_if_not_set,
     tier1,
     tier2,
-)
+    stubbed, tier4)
```
Comment (rochacbruno) [inline tests/foreman/api/test_repository.py]: wrap line here to keep the same pattern?

File: tests/foreman/api/test_contentview.py (hunk 1 above)
Comment (renzon) [inline tests/foreman/api/test_contentview.py]: Pycharm shortcut to push is Ctrl + Shift + K ;)

File: tests/foreman/api/test_repository.py (hunk 2 above)
Comment (renzon) [inline tests/foreman/api/test_repository.py]: Done

**Pull request merged**